src/containers/AdminCampaignEdit.jsx

```diff
@@ -502,31 +502,34 @@ class AdminCampaignEdit extends React.Component {
     );
 
     return (
-      <div
-        style={{
-          marginBottom: 15,
-          fontSize: 16
-        }}
-      >
-        {this.state.startingCampaign ? (
-          <div
-            style={{
-              color: theme.colors.gray,
-              fontWeight: 800
-            }}
-          >
-            <CircularProgress
-              size={0.5}
+      <div>
+        <h2>{this.props.campaignData.campaign.title}</h2>
```

Comment:
what happens with a new campaign?  Maybe we should condition the `h2` on a title exisitng?

Reply:
A new campaign appears to always contain the title property value 'New Campaign' as seen below:

![image](https://user-images.githubusercontent.com/12539396/80109848-dd8bb100-8532-11ea-845f-ac42cb16e5f2.png)

 Given that the value appeared to be guaranteed by the validation, I omitted any additional conditionality.